### ISSUE TITLE
Warnings when compiling

### ISSUE THREAD

**Issue by vkichline** [M5ez/M5ez#102]
When I compile with M5ez.h in PlatformIO, I get two warnings:
```
[...]\.platformio\lib\M5ez\src\M5ez.cpp:2238:2: warning: unknown escape sequence: '\|'
  "KB14|!?:;\\#$^&#SP#KB15|*()_-+=\|#Del#KB0|More#LCK:SYM|Lock#KB16|'\"`@%\\/#KB17|<>{}[]()#Done", //KB13
  ^
[...]\.platformio\lib\M5ez\src\M5ez.cpp:2241:2: warning: unknown escape sequence: '\#'
  "'#\"#`#@#%#/#\##Back",                //KB16
  ^
```

**Comment by vkichline** [comment]
Addressed in PR#104

**Comment by vkichline** [comment]
merged
